Tests/SourceKitLSPTests/CallHierarchyTests.swift

```diff
@@ -80,12 +81,12 @@ final class CallHierarchyTests: XCTestCase {
         kind: kind,
         tags: nil,
         detail: detail,
-        uri: location.uri,
+        uri: DocumentURI(URL(fileURLWithPath: resolveSymlinks(AbsolutePath(location.uri.pseudoPath)).pathString)),
```

Comment:
Do you think it would make sense to extract this snippet to a `nativeURI` computed property on `DocumentURI`. I think that would make it clearer why this pattern is needed. It might also be worth migrating your previous PRs to that computed property.

Reply:
I can see that as being a good approach.  Thanks for the idea!

Reply:
Replaced the appropriate cases.